### ISSUE TITLE
Failing for schemas with strange characters

### ISSUE THREAD

**Issue by alfredo-gimenez** [brianmhess/cassandra-loader#59]
Looks like it's not possible to use a schema with case-sensitive column names, e.g.

```
CREATE TABLE keyspace.table ("NaMe" text, "nAmE" text);
```

```
cassandra-loader -f myFileToLoad.csv -host 1.2.3.4 -schema 'keyspace.table("NaMe", "nAmE")'
```

The [regex check here](https://github.com/brianmhess/cassandra-loader/blob/a41cd688341440a12b6b2a5a8bbb71b5204f692e/src/main/java/com/datastax/loader/CqlDelimParser.java#L141) fails.

**Comment by brianmhess** [comment]
This works for me:
```
datastax@ubuntu:~/dev/cassandra-loader$ cqlsh -e 'create table ltest.test("NaMe" int primary key, "vAlUe" int)'
datastax@ubuntu:~/dev/cassandra-loader$ printf "1,1\n" | ./build/cassandra-loader -f stdin -host localhost -schema 'ltest.test("NaMe", "vAlUe")' 
*** Processing stdin
*** DONE: stdin  number of lines processed: 1 (1 inserted)
Lines Processed: 	0  Rate: 	0.0
datastax@ubuntu:~/dev/cassandra-loader$ cqlsh -e 'select * from ltest.test'
 NaMe | vAlUe
------+-------
    1 |     1

(1 rows)
```
What error are you seeing?

**Comment by alfredo-gimenez** [comment]
Sorry for the delay. Your test works fine for me too actually.

The schema that failed for me was actually pretty large and has some other weird characters (which I suspect are the issue).

```
CREATE TABLE cluster.data (
    node int,
    epoch bigint,
    "Auto_Shutdown" float,
    "BB_+1.1V_SB" float,
    "BB_+1.35_P1LV_AB" float,
    "BB_+1.35_P1LV_CD" float,
    "BB_+1.35_P2LV_AB" float,
    "BB_+1.35_P2LV_CD" float,
    "BB_+1.5_P1DDR_AB" float,
    "BB_+1.5_P1DDR_CD" float,
    "BB_+1.5_P2DDR_AB" float,
    "BB_+1.5_P2DDR_CD" float,
    "BB_1_1V_PCH" float,
    "BB_BMC_Temp" float,
    "BB_Inlet_Temp" float,
    "BB_P12V" float,
    "BB_P1_8V_AUX" float,
    "BB_P1_Vcc" float,
    "BB_P2_Vcc" float,
    "BB_P3_3V" float,
    "BB_P3_3V_STBY" float,
    "BB_P5V_STBY" float,
    "BMC_Health" float,
    "Button" float,
    "CATERR" float,
    "CPU_Missing" float,
    "DIMM_Thrm_Mrgn_1" float,
    "DIMM_Thrm_Mrgn_2" float,
    "DIMM_Thrm_Mrgn_3" float,
    "DIMM_Thrm_Mrgn_4" float,
    "IB_Temp" float,
    "IO_Mod_Presence" float,
    "IPMI_Watchdog" float,
    "LAN_NIC_Temp" float,
    "Mem_P1_Thrm_Trip" float,
    "Mem_P2_Thrm_Trip" float,
    "NM_Capabilities" float,
    "P1_DTS_Therm_Mgn" float,
    "P1_ERR2" float,
    "P1_MTT" float,
    "P1_Status" float,
    "P1_Therm_Ctrl_%" float,
    "P1_Therm_Margin" float,
    "P1_VRD_Hot" float,
    "P1_VR_Temp" float,
    "P2_DTS_Therm_Mgn" float,
    "P2_ERR2" float,
    "P2_MTT" float,
    "P2_Status" float,
    "P2_Therm_Ctrl_%" float,
    "P2_Therm_Margin" float,
    "P2_VRD_Hot" float,
    "Pwr_Unit_Status" float,
    "SAS_Mod_Presence" float,
    "SMI_TimeOut" float,
    "SSB_Temp" float,
    "SSB_Therm_Trip" float,
    "System_Event" float,
    "System_Event_Log" float,
    "VR_Watchdog" float,
    day int,
    hour int,
    idx bigint,
    jobid bigint,
    minute int,
    month int,
    nanosecond int,
    second int,
    year int,
    PRIMARY KEY (node, epoch)
)
```

The error is:

```
Exception in thread "main" java.util.concurrent.ExecutionException: java.text.ParseException: Badly formatted schema  cluster.data("idx", "jobid", "node", "epoch", "year", "month", "day", "hour", "minute", "second", "nanosecond", "Pwr_Unit_Status", "IPMI_Watchdog", "SMI_TimeOut", "System_Event_Log", "System_Event", "Button", "VR_Watchdog", "SSB_Therm_Trip", "IO_Mod_Presence", "SAS_Mod_Presence", "BMC_Health", "BB_Inlet_Temp", "SSB_Temp", "BB_BMC_Temp", "P1_VR_Temp", "IB_Temp", "LAN_NIC_Temp", "P1_Status", "P2_Status", "P1_Therm_Margin", "P2_Therm_Margin", "P1_Therm_Ctrl_%", "P2_Therm_Ctrl_%", "P1_ERR2", "P2_ERR2", "CATERR", "CPU_Missing", "P1_DTS_Therm_Mgn", "P2_DTS_Therm_Mgn", "P1_VRD_Hot", "P2_VRD_Hot", "DIMM_Thrm_Mrgn_1", "DIMM_Thrm_Mrgn_2", "DIMM_Thrm_Mrgn_3", "DIMM_Thrm_Mrgn_4", "Auto_Shutdown", "Mem_P1_Thrm_Trip", "Mem_P2_Thrm_Trip", "BB_P12V", "BB_P3_3V", "BB_P5V_STBY", "BB_P1_Vcc", "BB_P2_Vcc", "BB_+1.5_P1DDR_AB", "BB_+1.5_P1DDR_CD", "BB_+1.5_P2DDR_AB", "BB_+1.5_P2DDR_CD", "BB_P1_8V_AUX", "BB_+1.1V_SB", "BB_P3_3V_STBY", "BB_+1.35_P1LV_AB", "BB_+1.35_P1LV_CD", "BB_+1.35_P2LV_AB", "BB_+1.35_P2LV_CD", "BB_1_1V_PCH", "NM_Capabilities", "P1_MTT", "P2_MTT")
	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.util.concurrent.FutureTask.get(FutureTask.java:192)
	at com.datastax.loader.CqlDelimLoad.run(CqlDelimLoad.java:639)
	at com.datastax.loader.CqlDelimLoad.main(CqlDelimLoad.java:682)
Caused by: java.text.ParseException: Badly formatted schema  cluster.data("idx", "jobid", "node", "epoch", "year", "month", "day", "hour", "minute", "second", "nanosecond", "Pwr_Unit_Status", "IPMI_Watchdog", "SMI_TimeOut", "System_Event_Log", "System_Event", "Button", "VR_Watchdog", "SSB_Therm_Trip", "IO_Mod_Presence", "SAS_Mod_Presence", "BMC_Health", "BB_Inlet_Temp", "SSB_Temp", "BB_BMC_Temp", "P1_VR_Temp", "IB_Temp", "LAN_NIC_Temp", "P1_Status", "P2_Status", "P1_Therm_Margin", "P2_Therm_Margin", "P1_Therm_Ctrl_%", "P2_Therm_Ctrl_%", "P1_ERR2", "P2_ERR2", "CATERR", "CPU_Missing", "P1_DTS_Therm_Mgn", "P2_DTS_Therm_Mgn", "P1_VRD_Hot", "P2_VRD_Hot", "DIMM_Thrm_Mrgn_1", "DIMM_Thrm_Mrgn_2", "DIMM_Thrm_Mrgn_3", "DIMM_Thrm_Mrgn_4", "Auto_Shutdown", "Mem_P1_Thrm_Trip", "Mem_P2_Thrm_Trip", "BB_P12V", "BB_P3_3V", "BB_P5V_STBY", "BB_P1_Vcc", "BB_P2_Vcc", "BB_+1.5_P1DDR_AB", "BB_+1.5_P1DDR_CD", "BB_+1.5_P2DDR_AB", "BB_+1.5_P2DDR_CD", "BB_P1_8V_AUX", "BB_+1.1V_SB", "BB_P3_3V_STBY", "BB_+1.35_P1LV_AB", "BB_+1.35_P1LV_CD", "BB_+1.35_P2LV_AB", "BB_+1.35_P2LV_CD", "BB_1_1V_PCH", "NM_Capabilities", "P1_MTT", "P2_MTT")
	at com.datastax.loader.CqlDelimParser.processCqlSchema(CqlDelimParser.java:147)
	at com.datastax.loader.CqlDelimParser.<init>(CqlDelimParser.java:72)
	at com.datastax.loader.CqlDelimLoadTask.setup(CqlDelimLoadTask.java:182)
	at com.datastax.loader.CqlDelimLoadTask.call(CqlDelimLoadTask.java:145)
	at com.datastax.loader.CqlDelimLoadTask.call(CqlDelimLoadTask.java:53)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by brianmhess** [comment]
I believe this is now fixed with version v0.0.24.
